src/Storefront/Resources/app/storefront/src/main.js

```diff
@@ -161,6 +162,8 @@ PluginManager.register('Ellipsis', EllipsisPlugin, '[data-ellipsis]');
 PluginManager.register('SwagBlockLink', SwagBlockLink, '[href="#not-found"]');
 PluginManager.register('ClearInput', ClearInputPlugin, '[data-clear-input]');
 PluginManager.register('CmsGdprVideoElement', CmsGdprVideoElement, '[data-cms-gdpr-video-element]');
+PluginManager.register('ClearInput', ClearInputPlugin, '[data-clear-input]')
```

Comment:
Just noticed this is duplicated isn't it?
```suggestion
```

Reply:
Might've been a merge bug :) I'll do